gensim/models/keyedvectors.py

```diff
@@ -1695,6 +1697,44 @@ def intersect_word2vec_format(self, fname, lockf=0.0, binary=False, encoding='ut
             msg=f"merged {overlap_count} vectors into {self.vectors.shape} matrix from {fname}",
         )
 
+    def vectors_for_all(self, keys: Union[Iterable, Dictionary]) -> 'KeyedVectors':
+        """Produces vectors for all given keys.
```

Comment:
Docstrings in imperative mode (PEP8).

```suggestion
        """Produce vectors for all given keys.
```